### PR TITLE
Adding Slider component.

### LINKED ISSUE #175: Image Slider
**Is your feature request related to a problem? Please describe.**
We are getting request to add a Before/After slider functionality to pages.

image

**Describe the solution you'd like**
Build slider component to meet the increasing demand for this feature.

### PR DESCRIPTION
**Pull request by jedgar1mx**
## Changes
- Add `Slider` component

### CODE REVIEW

File: src/components/organisms/Slider/Slider.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  width: 100%;
+}
+
+slot[name='image-2'] {
+  position: absolute;
+  top: 0;
+  filter: drop-shadow(calc(var(--divider-width) * -1) 0 0 var(--divider-color));
+}
+
+.visually-hidden {
```
Comment (maxatdetroit) [inline src/components/organisms/Slider/Slider.css]: @jedgar1mx bootstrap has a utility class for this already. 

https://getbootstrap.com/docs/5.3/helpers/visually-hidden/

Is the override of the bootstrap CSS intentional here?

Comment (jedgar1mx) [reply src/components/organisms/Slider/Slider.css]: Nope, I just forgot to check bootstrap docs. Can you implement the bootstrap swap? Thanks

Comment (maxatdetroit) [reply src/components/organisms/Slider/Slider.css]: Yep, no problem.

**Comment by maxatdetroit** [comment]
@jedgar1mx I fixed the format/lints, updated the component to use the bootstrap default `.visually-hidden` and also renamed the component to `cod-img-slider` since `cod-slider` feels like it could be a bit too generic and may conflict with a more basic standalone "slider" component in the future.

**Pull request merged**